app/win.py

```diff
@@ -0,0 +1,47 @@
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills: list[str] = []

```

Comment:
The `skills` variable should be an attribute of the instance, not the class. This means that it should be defined inside the `__init__()` method. Check the second item in the checklist.